CIPs/CIP-23/CIP-23.md

```diff
@@ -0,0 +1,164 @@
+      "type": "object",
+      "properties": {
+        "protocol": {
+          "type": "string",
+		      "maxLength": 50
+        },
+        "host": {
+          "type": "string",
+		      "maxLength": 150
+        },
+        "id": {
+          "type": "string",
+		      "maxLength": 450
+        },
+        "claim": {
+          "type": "string",
+		      "maxLength": 450
+        },
+        "attestations": {
+          "type": "array"
+          "items": {
+            "type": "string",
+		      	"maxLength": 1000
+          }
+        }
+      },
+      "required": [
+        "protocol",
+        "host",
+        "id"
+      ]
+    }
+  }
+}
+```
+
+### Tags
+
+In the config of the *definition* defined above we can see a `tags` property. We simply apply the tags in the config to the `tags` property in the *metadata* of the *reference* Tile.
+
+### Example
+
+An example Crypto Accounts *reference* document that includes two Ethereum accounts and one Bitcoin account.
+
+```js
+const profile = await ceramic.createDocument('tile', {
+  metadata: {
+    schema: "<reference-schema-DocID>"
+    tags: ["SocialAccounts"]
+  },
+  content: [{
+		protocol: "https",
+		host: "https://twitter.com",
+		id: "https://twitter.com/marysmith",
+    // ID of tweet containing the user's DID
+		claim: "https://twitter.com/marysmith/status/1274020265417076736",
+		attestations: ["eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9..."]
+	}, {
+  	protocol: "https"
+		host: "https://github.com"
+		id: "https://github.com/marysmith"
+    // ID of Gist containing the user's DID
+		claim: "https://gist.github.com/marysmith/5c48debdb7089b3c8f86cca31739572c"
+		attestations: ["eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9..."]
+	}]
+})
+```
+
+
+## Suggested Usage
+
+It is recomended to always include both the `claim` and the `attestation` property. The claim is usually a way to verify that the social account actually want's to be associated with the users DID. The attestation is a [Verifiable Credential](https://github.com/decentralized-identity/did-jwt-vc) from a third party that this is the case. In some scenarious it might be impossible to provide a publicly verifiable `claim`. In these cases just an `attestation` will have to suffice.
+
+
+## Rationale
+
+**Decentralization & Trust:** Linked social account directories are data that needs to be globally-available, cross-chain, censorship-resistant, and live permissionlessly in the public domain (not on any single server). Additionally this information should be owned by a DID and will need to be updated from time to time. These requirements make Ceramic the most appropriate platform for publishing account links content.
+
+
+## Implementation
+
+- [**js-idx**](https://idx.xyz/): A javascript library for reading and writing data sets associated with a DID
+- [**js-idx-constants**](https://github.com/ceramicstudio/js-idx-constants): A javascript library to be used with js-idx which contains the Basic Profile definition.
```

Comment:
should be "which contains the <insert name we land on> definition." (Not Basic Profile)

Reply:
Err, yeah this has not been updated!